src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Server Data Encoding</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QComboBox" name="comboBox_encoding">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="toolTip">
```

Comment:
At present the trigger system is not going to fly with non-ASCII text - which is going to get in the way of anyone wanting to use the development code-base as anything other than a "technology preview" in those situations where they are using anything other than ASCII / ISO 8859-1 (or UTF-8 with _just the characters from those charsets_) - I still think a "this might make it look right but other things are not currently functional" for anything other than that first pair of encodings is a requirement at the moment! 😮